pkg/loop/reportingplugins/grpc_test.go

```diff
@@ -0,0 +1,69 @@
+package reportingplugins_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop"
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/test"
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop/reportingplugins"
+	"github.com/smartcontractkit/chainlink-relay/pkg/types"
+	"github.com/smartcontractkit/chainlink-relay/pkg/utils/tests"
+)
+
+func newStopCh(t *testing.T) <-chan struct{} {
```

Comment:
You could `t.CleanUp(func() { close(stopCh)})`, but then I suppose you need to `sync.Once` the close operation so maybe just making a mess :shrug: